src/print_zir.zig

```diff
@@ -550,7 +550,7 @@ const Writer = struct {
     fn writeElemTypeIndex(self: *Writer, stream: anytype, inst: Zir.Inst.Index) !void {
         const inst_data = self.code.instructions.items(.data)[inst].bin;
         try self.writeInstRef(stream, inst_data.lhs);
-        try stream.print(", {d})", .{inst_data.rhs});
+        try stream.print(", {})", .{inst_data.rhs});
```

Comment:
Not entirely sure what this should be. AFAIK `inst_data.rhs` here is an enum and the specifier was `{d}` and the function says it writes the element type index, so should this be `", {d})", .{@enumToInt(inst_data.rhs)}` instead?

Reply:
In this case `rhs` is used as a plain integer index so `@enumToInt` here is correct.

Reply:
This regressed at some point.